basics.js

```diff
@@ -4,11 +4,17 @@
 
 var pets = ['cat', 'dog', 'wombat'];
 
+
+for each (pets) {
+  "I have a " += pets;
+}
 ///// Question 2
 // Change my friend's hair color to "green", without modifying the line below.
 
 var friend = { name: 'Chris', age: 27, hairColor: 'blue' };
 
+friend = {hairColor: 'green'}
```

Comment:
Here you're completely overwriting the friend object, so the friend no longer has name and age properties.